experimental/dds/tree2/src/test/class-tree/treeApi.spec.ts

```diff
@@ -25,6 +26,38 @@ describe("treeApi", () => {
 		assert(nodeApi.is(root, Point));
 		assert(root instanceof Point);
 		assert(!nodeApi.is(root, schema.number));
-		assert(!nodeApi.is(5, schema.number));
+		assert(nodeApi.is(5, schema.number));
+		assert(!nodeApi.is(root, schema.number));
+		assert(!nodeApi.is(5, Point));
+	});
+	it("schema", () => {
```

Comment:
worth adding a test for `status`?

Reply:
I didn't change the implementing of that, so I don't see the need to add it in this change. Also turns out there are tests in src/test/simple-tree/node.spec.ts for src/class-tree/treeApi which cover it. I initially missed that file when making these changes since its in the wrong directory with the wrong name and the wrong test suite name.

Reply:
Those tests can get cleaned up/unified is a separate change